docs/functions.md

```diff
@@ -75,3 +75,45 @@ Which will produce the expression:
 ```hcl
 data.google_active_folder._parent_id.id.data
 ```
+
+## Experimental Functions
+
+These functions are experimental and some of them may only be available on
+specific contexts.
+
+### `tm_vendor(string) -> string`
+
+Receives a [Terraform module source](https://developer.hashicorp.com/terraform/language/modules/sources)
+as a parameter and returns the local path of the given module source after it is
+vendored. This function can only be used inside `generate_hcl` and
+`generate_file` blocks, it will work directly inside generated content
```

Comment:
maybe it should be said that it's not available when `context=root`?

Reply:
Indeed it should, thanks :100: